### PR TITLE
DDF-2033 fixed some configuration issues with the ldap bundles

### PR DESCRIPTION
**Pull request by stustison**
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@coyotesqrl @roelens8 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@kcwire
@stustison
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by roelens8** [comment]
Looks like we need to remove the KeystoreManagerTest.

File: platform/security/sts/security-sts-ldapclaimshandler/src/main/resources/OSGI-INF/metatype/metatype.xml [hunk 1]
```diff
@@ -70,7 +70,7 @@
 
 	</OCD>
 
-	<Designate factoryPid="Claims_Handler_Manager">
+	<Designate factoryPid="Claims_Handler_Manager" pid="Claims_Handler_Manager">
 	    <Object ocdref="Claims_Handler_Manager"/>
```
Comment (roelens8) [inline platform/security/sts/security-sts-ldapclaimshandler/src/main/resources/OSGI-INF/metatype/metatype.xml]: I don't think the pid is necessary? The factoryPid is the one referenced from the blueprint.

Comment (stustison) [inline platform/security/sts/security-sts-ldapclaimshandler/src/main/resources/OSGI-INF/metatype/metatype.xml]: The configuration does not appear in DDF without the pid.

**Comment by kcwire** [comment]
:+1

**Pull request merged**